src/main/java/org/mule/extension/api/runtime/MutableContentMetadata.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.runtime;
+
+/**
+ * Implementation of {@link ContentMetadata} which allows changing the
+ * {@link #outputContentType}
+ *
+ * @since 1.0
+ */
+public final class MutableContentMetadata implements ContentMetadata
```

Comment:
There are duplicated code on both versions of ContentMetadata. The immutable version can extend from the mutable